doc/builders/special/mkshell.section.md

```diff
@@ -0,0 +1,15 @@
+# pkgs.mkShell {#sec-pkgs-mkShell}
+
+`pkgs.mkShell` is a special kind of derivation that is only useful when using it combined with `nix-shell`. It will in fact fail to instantiate when invoked with `nix-build`.
+
+# Usage {#sec-pkgs-mkShell-usage}
```

Comment:
You cannot have two top-level headings of the same level, each file can contain only a single top-level section.

```suggestion
## Usage {#sec-pkgs-mkShell-usage}
```